ironfish/src/telemetry/telemetry.ts

```diff
@@ -228,17 +228,17 @@ export class Telemetry {
       return
     }
 
-    if (metric.fields.length === 0) {
-      throw new Error('Cannot submit metrics without fields')
-    }
-
     let tags = this.defaultTags
     if (metric.tags) {
       tags = tags.concat(metric.tags)
     }
 
     const fields = this.defaultFields.concat(metric.fields)
 
+    if (fields.length === 0) {
+      throw new Error('Cannot submit metrics without fields')
+    }
```

Comment:
Is there a measurement we want to send the default fields up without any other fields?

Reply:
Yes, the [`transaction_propagation`](https://github.com/iron-fish/ironfish/blob/8c6d2dd03ca9154aea996b5854f518301560935b/ironfish/src/telemetry/telemetry.ts#L404)